standalone/kubernetes/helm/squidex/values.yaml

```diff
@@ -0,0 +1,60 @@
+labels:
+service:
+  type: ClusterIP
+  port: 80
+deployment:
+  replicaCount: 1
+selectors:
+  component: squidex
+  partOf: ""
+  version: ""
+image:
+  repository: squidex/squidex
+  tag: latest
+  pullPolicy: IfNotPresent
+
+resources: {}
+nodeSelector: {}
+tolerations: []
+affinity: {}
+
+clusterSuffix: cluster.local
+
+ingress:
+  annotations: {}
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  hostName: squidex.local
+
+  tls: []
+  #  - secretName: chart-example-tls
+  #    hosts:
+  #      - chart-example.local
+
+env:
+  URLS__BASEURL: http://squidex.local/
```

Comment:
I have never used Helm but should these values not be configurable?

Reply:
the way it works is that this whole file exposes default values which you can (should) override

Reply:
Okay, but I think the default values for the followings fields do not make sense or are dangerous:

* URLS__BASEURL
* LETSENCRYPT_HOST
* LETSENCRYPT_EMAIL
* IDENTITY__ADMINEMAIL
* IDENTITY__ADMINPASSWORD

Can you mark these values as required?

Reply:
I believe so. Let me check what I can do

Reply:
Done. The error message goes one at a time and is not the best but it works:

![image](https://user-images.githubusercontent.com/16386764/64884181-09ef5e80-d62f-11e9-9c80-6479bd3b7977.png)

And I kept a sample-values file, to use it it's `helm template ./squidex -f sample-values.yaml` from the helm/ folder

With the samples the env variables look like so:
![image](https://user-images.githubusercontent.com/16386764/64884326-70747c80-d62f-11e9-8e44-7279250f6a7f.png)